src/config/constants.js

```diff
@@ -0,0 +1,9 @@
+
+module.exports = {
+  auth0: {
+    DOMAIN: process.env.AUTH0_DOMAIN,
```

Comment:
Another question I have is that every time I deploy... these values disappear in azure :( and I need to add it again, if I deploy a single function everything works as expected.

Anyone knows how to prevent that? Or how can we dynamically create those settings?

Reply:
Hi @crysfel, am not sure if these can be handled in the [local.settings.json](https://docs.microsoft.com/en-us/azure/azure-functions/functions-run-local#local-settings-file)

Reply:
you could set these variables in the serverless.yml